### ISSUE TITLE
user being fetched only after browser tabs changed by hand

### ISSUE THREAD

**Issue by Solemensis** [nuxt-modules/supabase#95]
after login, the user value is shown on the page, but immedietaly gone after refreshing page or changing route. and it's being fetched back again only after browser tabs changed. And if you reload page or go to another route, it'll hide again. You need to change tabs again to show it. It doesn't happen this way on localhost. Only on netlify etc. 

The hosted buggy website:
https://luminous-haupia-12c8f2.netlify.app/login

try to login with:
mail: myauthtest23@outlook.com // pass: 123321

or create your own account and login with it.

After login, there should be uservalue written on page if there's a user. So after login, it shows up, but gone after route change or website reload. And, it comes back after tab changes.

I've recorded a vid:
https://www.youtube.com/watch?v=SOTzkS08YIs&ab_channel=Clarity

Here's the repo of this to reproduce:
https://github.com/Solemensis/supabase-nuxt-login_bug

**Comment by CodyBontecou** [comment]
I'm also running into this bug. I've tried `nuxt: 3.0.0-rc.12` and see the same results.

**Comment by larbish** [comment]
Closing in favor of #101 (merging all Netlify issues)

**Comment by larbish** [comment]
@CodyBontecou I'm not reproducing in local with [this repo](https://github.com/Solemensis/supabase-nuxt-login_bug). Do you ?

**Comment by Solemensis** [comment]
> I'm also running into this bug. I've tried `nuxt: 3.0.0-rc.12` and see the same results.
> 
> I've noticed you attached the netlify tag to this. I'm seeing this issue during local development.

Yeah interesting, local works for me, started working only after latest npm package of supabase/nuxt. Host problem still goes tho.

**Comment by anderslutzhoft** [comment]
I'm facing a similar issue, but on local dev as well #144
